winui/Time-Picker/Time-Restriction.md

```diff
@@ -25,7 +25,7 @@ sfTimePicker.MaxTime = new DateTimeOffset(new DateTime(DateTime.Now.Year, DateTi
 {% endhighlight  %}
 {% endtabs %}
 
-![change-minimum-and-maximum-times-in-winui-time-picker](Getting-Started_images/change-minimum-and-maximum-times-in-winui-time-picker.png)
+![Change minimum and maximum times in WinUI TimePicker](getting-started_images/winui-time-picker-change-minimum-and-maximum-times.png)
```

Comment:
This files too, revert your image name changes.

Reply:
Changed